### PR TITLE
SUP-2582, Upcoming SRMs should be ordered by most recent date

### PR DESCRIPTION
**Pull request by vikasrohit**
-- Fixed sorting in dashboard (ascending by registrationStartAt)
-- Fixed sorting in My SRMs#Upcoming (ascending by codingStartAt)
-- Fixed sorting in My SRMs#Past (ascending by codingEndAt)

@parthshah @oyeanuj fyi

### CODE REVIEW

File: app/my-dashboard/srms/srms.controller.js [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
     function getSRMs() {
       var params = {
-        filter: 'status=future',
+        filter: 'status=future&orderBy=registrationStartAt',
```
Comment (parthshah) [inline app/my-dashboard/srms/srms.controller.js]: 'orderBy' is *not* a filter. It should be passed as a separate param like 'limit'

File: app/my-srms/my-srms.controller.js [hunk 2]
```diff
@@ -52,7 +52,7 @@
 
     function changeFilter(filter) {
       vm.statusFilter = filter;
-      vm.orderBy = filter === 'future'? 'startDate': 'endDate';
+      vm.orderBy = filter === 'future'? 'codingStartAt': 'codingEndAt';
```
Comment (parthshah) [inline app/my-srms/my-srms.controller.js]: Please add some comments stating why we are using one over the other. Thanks

**Comment by vikasrohit** [comment]
Incorporated changes. Please review.

**Comment by parthshah** [comment]
Looks good. Thanks!

**Pull request merged**